client/app/components/search/Search.react.js

```diff
@@ -51,6 +59,9 @@ var Search = React.createClass({
     //Send value to server before erasing it!
     WikiPoetryActionCreators.submitSearch(search);
     WikiPoetryActionCreators.getArticleContent(this.state.type, search);
+    this.setState({
```

Comment:
I don't think we should do this. I've read a couple of places that setting state in render is a big anti-pattern. https://facebook.github.io/react/docs/component-specs.html#render

I'd be better if state was set in another lifecycle hook and I'm guessing the most preferred way would be to have an action set the state in another lifecycle hook.